### ISSUE TITLE
GitHub Ruleset awareness

### ISSUE THREAD

**Issue by abacchilb** [Legit-Labs/legitify#309]
### TL;DR

A nice feature GitHub launched is the ability to apply "rulesets" to your organization at scale. This allows for branch protection for all repositories if you want. I set this up- but I am not seeing the result reflected in my Legitify analysis result.

### Additional information

https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/available-rules-for-rulesets#additional-settings

When I create a ruleset - I'd love to have Legitify ack the protections implemented. I'm guessing that there is a different attribute applied and not technically "branch protection" when I set ruleset to "Require a pull request before merging", or "Required approvals: 1" etc.

**Comment by abacchilb** [comment]
I may be mistaken- It seems that the policy was evaluated correctly
